runtime/libs/circle-schema/include/circle_traininfo.fbs

```diff
@@ -1,5 +1,3 @@
-// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/5dd9665b3dab4652a0afed50da86bf6964a41bdd/spec/circle_training.fbs
-
```

Comment:
related inner source issue : https://github.sec.samsung.net/one-project/circle-x/issues/57 

Now, inner source doesn't hold `circle_traininfo.fbs`. The schema file will be maintained in public.
So, Let's remove this line that points a innersource commit number.